docs/docs/components/guidelines/README.md

```diff
@@ -258,16 +258,16 @@ out](https://pipedream.com/community/c/dev/11).
 
 ##### Prop Definitions
 
-Whenever possible, reuse existing [prop
-definitions](https://github.com/PipedreamHQ/pipedream/blob/master/COMPONENT-API.md#prop-definitions-example).
+Whenever possible, reuse existing [prop definitions](../api/#prop-definitions-example).
```

Comment:
Relative links should be relative to the docs root for clarity, so this is preferred:

```
[prop definitions](/components/api/#prop-definitions-example)
```